### PR TITLE
Entities editor fixes

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
- Fix display of large negative coordinates and angles (numbers were being obscured by spin buttons)
- Fix fonts on OSX (text was serif, letters were displaying instead of glyphs)
- Fix horizontal scrollbar appearing in entity properties tab on OSX (it shouldn't)
- Fix spin box buttons not appearing on OSX
- Fix right margin of entities list

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9119b8271883e64900ba2b0daed810e9ec8d5e1a available at: https://gist.github.com/e049ac9c52a895aa508685a02204a83d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7601.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7601.dmg

**Comment by kevinmthomas-carpool** [comment]
## QA Test
**PR Build:** 7601
**Testing Architecture:** 64-bit PC Desktop & 64-bit MAC Laptop
**Testing OS:** Windows 10 & OS X
**Testing Locations:** sandbox, playa, cellscience, forum
**Smoke Test Results:** successful PC/MAC

                                  | playa            | cellscience     | forum   
----------------------------------|------------------|-----------------|--------
Render Rate:                      |63/28             |66/38            |63/26            
Present Rate:                     |60/60             |60/60            |60/60           
Simrate:                          |62/27             |65/38            |62/26            
Render Stats Num Buffers:         |3240/3830         |4141/2520        |4736/5516              
Render Stats gpu::Buffer Memory:  |60.5Mb/105.7Mb    |118.3Mb/37.0Mb   |71.7/148.5Mb              
Render Stats Num Textures:        |392/438           |210/197          |508/343              
Render Stats gpu::Texture Memory: |1050.4Mb/1930.4Mb |1064.6Mb/337.4Mb |983.0Mb/1340.2Mb              
Render Stats Triangles:           |651k/714k         |480k/569k        |116k/596k             
Render Stats Drawcalls:           |4499/9845         |3353/1614        |1721/294             
Render Stats Items:               |212/334           |796/784          |268/179              

**sandbox ‘createtestblocks.js’ Result:** successful PC/MAC
**sandbox Import FBX Result:** successful PC/MAC
**sandbox ‘harmonicOscillator.js’ Result:** successful PC/MAC
**sandbox Echo Server Audio Result:** successful PC/MAC
**sandbox ‘reverbTest.js’ Result:** successful PC/MAC
**Additional Test Results Requested:** (none requested)
**Successful Logout/Crash (If  A Crash Occurs, Attach ‘hifi-log.txt’):** Crash Windows PR Build, attached .dmp and .log file for further examination.  MAC PR Build had a successful Interface logout no crash.  Attached MAC PR build screenshots showing expected behavior in edit.js windows.  
[attachments.zip](https://github.com/highfidelity/hifi/files/207385/attachments.zip)

**Comment by ctrlaltdavid** [comment]
The HTML/CSS fixes are good, then. The crash at shutdown is almost certainly unrelated.

File: examples/html/edit-style.css [hunk 1]
```diff
@@ -10,43 +10,50 @@
 
 @font-face {
     font-family: Raleway-Regular;
-    src: url(../../resources/fonts/Raleway-Regular.ttf),  /* Production */
+    src: url(../../resources/fonts/Raleway-Regular.ttf),  /* Windows production */
+         url(../../fonts/Raleway-Regular.ttf),  /* OSX production */
          url(../../interface/resources/fonts/Raleway-Regular.ttf);  /* Development */
```
Comment (jherico) [inline examples/html/edit-style.css]: so, once you build, the resources are copied to the build directory.  so if you want the source directory copy to take precedency, do you need it to come first?

File: examples/html/edit-style.css [hunk 2]
```diff
@@ -10,43 +10,50 @@
 
 @font-face {
     font-family: Raleway-Regular;
-    src: url(../../resources/fonts/Raleway-Regular.ttf),  /* Production */
+    src: url(../../resources/fonts/Raleway-Regular.ttf),  /* Windows production */
```
Comment (jherico) [inline examples/html/edit-style.css]: we should expose the resources directory to the QML context as a global, so we can just say `GlobalResourcesDirectory + '/fonts/Raleway-Regular.ttf'`

File: examples/html/edit-style.css (hunk 1 above)
Comment (ctrlaltdavid) [inline examples/html/edit-style.css]: No, the "development" path is for when running the script from \HiFi\examples source directory. I'll update the comment to clarify.

File: examples/html/edit-style.css (hunk 2 above)
Comment (ctrlaltdavid) [inline examples/html/edit-style.css]: Unfortunately, CSS doesn't do expressions.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 86d314f31a3d90c0bf19cdd3938519bc2e3e4130 available at: https://gist.github.com/e049ac9c52a895aa508685a02204a83d<br/><br/>Installers available for download at: <br/>Windows: http://s3-us-west-1.amazonaws.com/hifi-pr-builds/HighFidelity-Alpha-PR7601.exe<br/>OS X: http://s3-us-west-1.amazonaws.com/hifi-pr-builds/HighFidelity-Alpha-PR7601.dmg

**Pull request merged**